.github/actions/gomodtidy/Dockerfile

```diff
@@ -2,4 +2,6 @@ FROM golang:1.21-alpine
 
 COPY entrypoint.sh /entrypoint.sh
 
+USER nonroot
```

Comment:
This is failing this action now.

> docker: Error response from daemon: unable to find user nonroot: no matching entries in passwd file.

Reply:
@prestonvanloon fixed, re-review please

Reply:
```
cp: can't create 'go.mod.orig': Permission denied
```
Please run CI locally using [act](https://github.com/nektos/act)